roles/network_plugin/calico/defaults/main.yml

```diff
@@ -29,6 +29,7 @@ calicoctl_memory_limit: 170M
 calicoctl_cpu_limit: 100m
 calicoctl_memory_requests: 32M
 calicoctl_cpu_requests: 250m
+calico_felix_chaininsertmode: Insert
```

Comment:
Calico [docs](https://github.com/projectcalico/calico/blob/master/v2.1/reference/felix/configuration.md) says the default value is `insert`, not sure if case sensitive.

Reply:
Hmm, I tested with `Insert` and `Append`, because that's what was on https://docs.projectcalico.org/v3.9/reference/felix/configuration , but I'm not really a calico expert. I think they have [a test for case insensitivity](https://github.com/projectcalico/felix/blob/dc02bc8323b2ce9ec5bda04c89f9179083fcc4ae/config/config_params_test.go#L203-L205) too.